--- a/build.rs
+++ b/build.rs
@@ -105,6 +105,11 @@

     println!(
         "cargo:rustc-link-search=native={}",
+        cmake_dir.join("lib64").display()
+    );
+
+    println!(
+        "cargo:rustc-link-search=native={}",
         cmake_dir.join("bin").display()
     );
 }
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -2,6 +2,7 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(improper_ctypes)]
+#![allow(unpredictable_function_pointer_comparisons)]

 pub mod built_info {
     include!(concat!(env!("OUT_DIR"), "/built.rs"));
